bench/workgen/runner/example_dynamic_tables.py

```diff
@@ -37,6 +37,22 @@
 from workgen import *
 
 
+class ThreadWithReturnValue(pythread.Thread):
+
+    def __init__(self, group=None, target=None, name=None,
+                 args=(), kwargs={}, Verbose=None):
+        pythread.Thread.__init__(self, group, target, name, args, kwargs)
```

Comment:
Should we add `Verbose` as the last argument?

Reply:
I think we can remove Verbose argument - it is not being used and I don't think it is part of the main Threading class as well.

Reply:
updated here: [5e2ca4d](https://github.com/wiredtiger/wiredtiger/pull/8544/commits/5e2ca4d42b8e13aea7e22422b3165f7e2b4b759e)